glide.yaml

```diff
@@ -19,7 +19,7 @@ import:
 - package: github.com/giantswarm/awstpr
   version: master
 - package: github.com/giantswarm/k8scloudconfig
-  version: master
+  version: v3.0.0-coredns-1.0.4
```

Comment:
Shouldn't this be master?

Reply:
Oof noticed you haven't asked for review! Sorry ignore!

Reply:
I'm just testing stuff. After testing, I'll change it to the master and update vendor again